.gitignore

```diff
@@ -7,5 +7,4 @@ kubectl-directpv
 !kubectl-directpv/
 vdb.xml
 drives.yaml
-kubectl-*
```

Comment:
we have `kubectl-directpv` in `.gitignore`, so the local binaries will be ignored.

we need to remove this because any new files in `kubectl-directpv/` are not getting committed.